website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -1,203 +1,84 @@
-   <TabItem label="Community Edition" value="community">
-
-This command downloads the image and starts Appsmith:
-
-```bash
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" --pull always appsmith/appsmith-ce
-```
-
-   </TabItem>
-   <TabItem label="Business Edition" value="business">
-
-This command downloads the image and starts Appsmith:
-
-```bash
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" --pull always appsmith/appsmith-ee
-```
-
-   </TabItem>
-</Tabs>
-
-### Updating Appsmith
-
-<Tabs groupId="editions">
-   <TabItem label="Community Edition" value="community">
-
-To update Appsmith manually, go to the root directory of the installation and run the following commands:
-
-```bash
-docker rmi appsmith/appsmith-ce -f
-docker pull appsmith/appsmith-ce
-docker rm -f appsmith
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" appsmith/appsmith-ce
-```
-
-   </TabItem>
-   <TabItem label="Business Edition" value="business">
-
-To update Appsmith manually, go to the root directory of the installation and run the following commands:
+2. To start the Docker container, use the following command. You may have to run the command with `sudo` if you don't have permission to execute `docker-compose`. The below command downloads the Docker image if not present locally.
 
 ```bash
-docker rmi appsmith/appsmith-ee -f
-docker pull appsmith/appsmith-ee
-docker rm -f appsmith
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" appsmith/appsmith-ee
+docker-compose up -d
 ```
 
-   </TabItem>
-</Tabs>
-
-Once the download is complete, the server should be up soon. You can verify that it's working correctly in the [Docker logs](#docker-logs).
-
-## Docker logs
-
-Once the Appsmith container is ready, the message "Appsmith is Running!" is displayed in the Docker logs.
+The server deploys Appsmith at startup, and you can access it at [http://localhost](http://localhost). You can also check the message "Appsmith is Running!" displayed in docker logs by using `docker logs -f appsmith` command.
 
 ![Appsmith is running message](/img/InstallationGuides__Docker__AppsmithRunningMessage.png)
 
-You can follow the logs with the following command:
-
-```bash
-docker logs -f appsmith
-```
-
-**Server logs** are stored in `stacks/logs/backend/backend.log`. 
-
-Logs can assist when debugging or analyzing a problem because they offer valuable information about what went wrong. For more information, see [How to get container logs](/learning-and-resources/how-to-guides/how-to-get-container-logs).
-
-
-## Restarting containers
-
-If the Docker containers are failing to restart, download, and run the [`restart-container.sh`](/img/restart-container.sh) script linked below to bring them up:
-
-Copy the script (`restart-container.sh`) to the installation folder and make it executable:
-
-```bash
-chmod +x restart-container.sh
-./restart-container.sh
-```
+:::info important
```

Comment:
I feel all the steps should be on the respective tabs so that flow isn't broken.

Reply:
As discussed- I've duplicated them. However, we can discuss and confirm whether we should redirect the users to the customer portal for signing up and installing the business edition. In which case, the docs will only have instructions for installing community edition.

Reply:
As discussed, we now have the tab urls.